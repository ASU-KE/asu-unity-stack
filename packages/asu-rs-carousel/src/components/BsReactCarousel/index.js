import BsReactCarousel from './BsReactCarousel';

export default BsReactCarousel;
