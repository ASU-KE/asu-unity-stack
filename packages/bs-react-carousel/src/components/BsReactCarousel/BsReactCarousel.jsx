import React from 'react';
import PropTypes from 'prop-types';
//import { Test } from './BsReactCarousel.styles';
import { UncontrolledCarousel, Util} from 'reactstrap';
import bootstrap from '../../styles/bs4/css/bootstrap-asu.min.css';

// Needs to be set for CSS modules.
// See https://github.com/reactstrap/reactstrap/issues/1049
Util.setGlobalCssModule(bootstrap);

const BsReactCarousel = (props) => (
  <UncontrolledCarousel cssModule={bootstrap} items={props.items}/>
);

BsReactCarousel.propTypes = {
  items: PropTypes.arrayOf(PropTypes.object)
};

BsReactCarousel.defaultProps = {
};

export default BsReactCarousel;
