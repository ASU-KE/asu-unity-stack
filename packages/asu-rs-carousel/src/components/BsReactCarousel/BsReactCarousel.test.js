import React from 'react'
import { render, fireEvent} from '@testing-library/react'
import BsReactCarousel from './BsReactCarousel'



describe('The BsReactCarousel', () => {
    it('renders the component', () => {

      const container = render(<BsReactCarousel  {...{items: [{}]}} />);
            expect(container.firstChild).toMatchSnapshot()
    })
});
