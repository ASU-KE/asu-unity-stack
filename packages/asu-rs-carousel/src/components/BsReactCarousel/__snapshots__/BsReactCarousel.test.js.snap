// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The BsReactCarousel renders the component 1`] = `undefined`;
